arthur_bench/server/run_server.py

```diff
@@ -222,7 +223,9 @@ def run():
         persist_usage_data(False)
         return
 
-    client = LocalBenchClient(args.directory)
+    # TODO(zfry): think about how to do this better
+    conf = LocalFSClientConfig(root_dir=args.directory)
+    client = LocalBenchClient(conf)
```

Comment:
this (and the same thing im doing in the tests) are what I'm not sure about 

open to ideas/suggestions